### PR TITLE
Run scripts directly instead of building

### PR DESCRIPTION
**Pull request by jasonaowen**
The scripts in this repo are primarily intended for developers to run; they are not deployed anywhere, and in particular do not need to be run outside of npm or in a context without dev dependencies installed.

Switch from the two-step build & run approach to running the TypeScript files directly via ts-node.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,10 @@
...
   "main": "dist/generateBaseFieldsInserts.js",
   "scripts": {
-    "build": "npm run cleanup && npm run build:node",
-    "build:node": "tsc -p tsconfig.json",
-    "cleanup": "rimraf dist",
     "lint": "eslint ./src --ext .ts",
     "test": "echo \"no tests\" && exit 1",
-    "generateBaseFieldsInserts": "node dist/generateBaseFieldsInserts.js",
-    "generateApplicationFormJson": "node dist/generateApplicationFormJson.js"
+    "generateBaseFieldsInserts": "ts-node src/generateBaseFieldsInserts.ts",
```
Comment (slifty) [inline package.json:9]: Sanity check: do we need to specify the file extensions?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -13,4 +13,4 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Build
-        run: npm run build
+        run: tsc -p tsconfig.json
```
Comment (slifty) [inline .github/workflows/build.yml:16]: Would it be appropriate to rename this workflow to something else or is build still the best verb for what this CI is doing?

**Review by slifty — APPROVED** [review]
Two small things for your consideration but otherwise this makes sense.

(also I see we have lint warnings, we should probably resolve those -- maybe simply allowing console given the nature of this repo.)

File: package.json (hunk 1 above)
Comment (jasonaowen) [reply package.json:9]: Oh! Excellent catch, we do not. That never even occurred to me!

File: .github/workflows/build.yml (hunk 2 above)
Comment (jasonaowen) [reply .github/workflows/build.yml:16]: I gave that some brief thought while I was working on this PR, and my initial lazy response was "no". But the fact that you also raised that question makes me revisit! Options include "Check types" and "Compile"; any others spring to mind?

Somewhat relevant is the [documentation on the `--no-emit` option](https://www.typescriptlang.org/tsconfig/#noEmit):

> Do not emit compiler output files like JavaScript source code, source-maps or declarations. [...] You can then use TypeScript [...] as a source code type-checker.

**Comment by jasonaowen** [comment]
> (also I see we have lint warnings, we should probably resolve those -- maybe simply allowing console given the nature of this repo.)

We definitely should, one way or another, but outside of this PR. I'm planning to introduce some logging, but that'll be a little bit of a conversation that we don't need to start here.

File: .github/workflows/build.yml (hunk 2 above)
Comment (jasonaowen) [reply .github/workflows/build.yml:16]: I went with "check types"; let me know if you have strong opinions in another direction!

**Comment by jasonaowen** [comment]
Updated to address @slifty's comments - thanks for the quick review!

@bickelj I do want to wait for your review, as you're the one who has been running these scripts in practice, and in particular I don't want to break your existing workflow without your consent.

**Review by bickelj — APPROVED** [review]
This is much cleaner, thank you!

It doesn't block my flow at the moment because I am still working in an admittedly awkward long-lived off-to-the-side branch.

A pino logger is welcome.

**Pull request merged**